timeout_tools/cli.py

```diff
@@ -158,9 +162,13 @@ def python_setup_func(args):
 
 
 def python_setup(app, branch, python_version):
+    try:
+        check_python_version_installed(python_version)
```

Comment:
I feel the logic shouldn't be inside the check function, ie if the check returns false here then call the install here

Reply:
certainly makes the tests easier